### PR TITLE
ZER-6 Create calclator value entry page

### PR DESCRIPTION
**Pull request by haliapats**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -38,7 +38,6 @@ en:
         childs_birthday_label: "Child's birthday"
         baby_weight_label: "Baby weight"
       want_recomentation_label: "Yes, I want to receive email messages with recommendations"
-      calculate_result_button: "CALCULATE RESULT"
```
[the remark this answers is not in the document]
Comment (haliapats) [reply config/locales/en.yml:41]: I had to rename the button to Ukrainian, so I localized it in uk.yml

File: app/javascript/ajax/result_button.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+$(document).on('turbolinks:load', function() {
+  const form = document.getElementById('form');
+
+  form.addEventListener('submit', function(e) {
+
+    e.preventDefault();
+    
+    const formData = {
+      childs_birthday: $("#birth").val(),
+      baby_weight: $("#weight").val()
```
Comment (maksymst) [inline app/javascript/ajax/result_button.js:10]: this is not required

File: app/views/calculators/show.html.erb [hunk 3]
```diff
@@ -5,19 +5,15 @@
...
+            <input id="birth" type="text" class="col datepick" />
           </label>
           <label for="" class="text-center">
             <%= t '.form.baby_weight_label' %>
-            <input type="text" class="col datepick" />
-          </label>
-          <label for="" class="text-center">
-            <%= t '.form.childs_birthday_label' %>
-            <input type="text" class="col datepick" />
+            <input id="weight" type="text" class="col datepick" />
```
Comment (maksymst) [inline app/views/calculators/show.html.erb:16]: remove this field

File: app/views/calculators/show.html.erb [hunk 4]
```diff
@@ -32,7 +28,7 @@
             <%= t '.want_recomentation_label' %>
           </label>
         </div>
-        <button type="submit" class="btn btn-success ml-3 mt-4">
+        <button type="submit" id = "result_button" class="btn btn-success ml-3 mt-4">
```
Comment (maksymst) [inline app/views/calculators/show.html.erb:31]: do you need this change?

File: config/locales/en.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (maksymst) [reply config/locales/en.yml:41]: I understand, but shouldn't it be also in en.yml?

**Review by maksymst — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (haliapats) [reply config/locales/en.yml]: Ok, I understood, added back
Thanks)

**Review by maksymst — APPROVED** [review]

**Pull request merged**